specifyweb/frontend/js_src/lib/components/Interactions/InteractionsDialog.tsx

```diff
@@ -34,8 +34,8 @@ export const interactionWorkflowTables: Partial<
   Loan: 'CollectionObject',
   Gift: 'CollectionObject',
   Disposal: 'CollectionObject',
+  ExchangeOut: 'Preparation',
```

Comment:
Were the values of this object going to be used anywhere? 
Currently, only the key values are used. 

With little effort this can functionally be replaced with a new constant for values in `InteractionsWithPreps` defined the same way as `interactionPrepTables`.